marc_to_solr/lib/princeton_marc.rb

```diff
@@ -628,132 +628,170 @@ def scsb_doc?(record_id)
 
 # holding block json hash keyed on mfhd id including location, library, call number, shelving title,
 # location note, location has, location has (current), indexes, and supplements
-# pulls from mfhd 852, 866, 867, and 868
-# assumes exactly 1 852 is present per mfhd (it saves the last 852 it finds)
+# pulls from mfhd AVA, 866, 867, and 868
+# assumes exactly 1 AVA is present per mfhd (it saves the last AVA it finds)
```

Comment:
@jrgriffiniii I havent looked the whole Pr but when we index from the mrc files that we export from alma they include the enrichment fields 852. They dont include AVA. The AVA is a tag that comes back only through the Alma API call. Please reverse the changes in princeton_marc file.